projects/go-lib/src/lib/components/go-header-bar/go-header-bar.component.scss

```diff
@@ -0,0 +1,39 @@
+@import "../../../styles/_variables.scss";
+.go-header-bar {
+  margin: -2rem;
```

Comment:
We can use the `$column-gutter--double` variable for this, you'll still need to make it negative though.